public/src/components/channelManagement/epicTests/epicVariantPreview.tsx

```diff
@@ -35,11 +121,49 @@ const buildProps = (variant: EpicVariant): EpicProps => ({
     heading: variant.heading,
     paragraphs: variant.paragraphs,
     highlightedText: variant.highlightedText,
-    showTicker: false,
     cta: variant.cta,
+
+    secondaryCta: variant.secondaryCta,
+    showReminderFields: {
+      reminderCta: 'Remind me in October',
+      reminderPeriod: '2021-10-01',
+      reminderLabel: 'October',
+    },
+
     separateArticleCount: variant.separateArticleCount,
-    showSignInLink: variant.showSignInLink,
+
+    // There's some weird stuff going on around the sign-in link
+    // - we have an 'enable sign-in link' checkbox in RRCP
+    // - but no indication that the checkbox setting gets set in variant data
+    // - for now, always show the link for the purposes of LIVE preview
```

Comment:
I'm not sure what you mean - this seems to work fine -
![Screen Shot 2022-05-18 at 08 22 35](https://user-images.githubusercontent.com/1513454/168981117-19050592-a81a-48ed-a227-cf8d58baa0f9.png)

This should be `showSignInLink: variant.showSignInLink`

Reply:
Assuming Marketing colleagues have chosen not to set the attribute for any of their tests currently showing in RRCP LIVE (which led to my misunderstanding), code updated as suggested